antarest/study/storage/rawstudy/ini_reader.py

```diff
@@ -166,12 +166,13 @@ def __repr__(self) -> str:  # pragma: no cover
 
     def read(self, path: t.Any, **kwargs: t.Any) -> JSON:
         if isinstance(path, (Path, str)):
+            sanitized_path = Path(path).resolve()
```

Comment:
I don't think resolves are enough to check our input paths, although it could satisfy sonar.
I think instead we should check that they don't lead to unsecure paths for us like paths that go in undesired folders.

We need to think a little about this!